src/Sulu/Bundle/WebsiteBundle/Command/DumpSitemapCommand.php

```diff
@@ -93,15 +106,17 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $output->writeln('Start dumping "sitemap.xml" files:');
 
-        $webspaceManager = $this->getContainer()->get('sulu_core.webspace.webspace_manager');
-
-        /** @var PortalInformation $portalInformation */
-        foreach ($webspaceManager->getPortalInformations($this->environment) as $portalInformation) {
-            if (RequestAnalyzerInterface::MATCH_TYPE_FULL !== $portalInformation->getType()) {
```

Comment:
Why is this check not enough anymore? Why do we need 3 nested foreach loops instead?